README.org

```diff
@@ -58,6 +58,33 @@ configuration file:
 You can also decide to only activate beginend for some of its
 supported major modes (e.g., through ~beginend-dired-mode~).
 
+*** init.el example configuration
+
+By default, Beginend adds the ~" be"~ lighter to the modeline. You can use [[https://github.com/myrjola/diminish.el][diminish.el]] or [[https://savannah.nongnu.org/projects/delight][delight.el]] to turn this off for every minor mode Beginend defines by looping over the ~beginend-modes~ variable. Note that you will /also/ need to "turn off the lights" for ~beginend-global-mode~, which is not part of this list.
+
+In the following snippets, you can replace ~delight~ with ~diminish~ for the same results.
+
+Vanilla Emacs ~init.el~:
+
+#+BEGIN_SRC emacs-lisp
+(require 'beginend)
+(dolist (mode (cons 'beginend-global-mode (mapcar 'cdr beginend-modes)))
```

Comment:
`'cdr` => `#'cdr`

Reply:
Can you give me a short hint why that's necessary? (It works without, too, and I always get the prefixes mixed up 😅)

Reply:
Sure: https://endlessparentheses.com/get-in-the-habit-of-using-sharp-quote.html

Reply:
Thanks, the blog is a great resource!